specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4238,6 +4268,24 @@
       "type": "boolean",
       "in": "query",
       "x-ms-parameter-location": "method"
+    },
+    "detectionModel": {
+      "name": "detectionModel",
+      "description": "Name of detection model. Detection model is used to detect face from an uploaded image. A detection model name can be provided when performing Face - Detect or (Large)FaceList - AddFace or (Large)PersonGroupPerson - AddFace. The default value is 'detection_01', if latest model needed, please explicitly specify the model you need.",
```

Comment:
[Nit] `Detection model is used to detect face from an uploaded image.` -> `Detection model is used to detect faces from images.`

Reply:
Done.